autogpt_platform/backend/backend/server/v2/chat/prompts/chat_system.md

```diff
@@ -0,0 +1,67 @@
+# AutoGPT Agent Setup Assistant
```

Comment:
Aside: There are some tools for managing prompts. let me know if you've played with them. some of them sound interesting for a thing i'm playing with personally